third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -16,14 +16,15 @@ typedef std::function<SmallVector<Value>(Location, ConversionPatternRewriter &,
     ConverterT;
 
 namespace {
-/* ----- FP8E5M2 ------ */
+// ROCM utility functions for data type conversion
 /* ----- FP8E5M2 ------ */
 // This data-type is the standard FP8E5M2 format
-#ifdef USE_ROCM
-
-static SmallVector<Value> Fp16_to_Fp8E5M2(Location loc,
-                                          ConversionPatternRewriter &rewriter,
-                                          const SmallVector<Value> &v) {
+// This function is not fully functional for the denorm values
+// And currently it is not used as we don't have HW native support on
+// MI300. We will rewrite it if upstream adds the support for A100
```

Comment:
.. adds the support for MI300.

Actually I'd suggest to reorganize the comment here a bit.

"This data type is the standard FP8E5M2 format. NVIDIA GPU supports it natively but we don't have hardware native support on MI300.

So these helper functions are not fully functional for the denorm values. We need rewrite it if we want to emulate this data type on MI300."

Reply:
Done.